code/game/mecha/mecha.dm

```diff
@@ -40,6 +40,7 @@
 	var/emagged = FALSE
 	var/frozen = FALSE
 	var/repairing = FALSE
+	var/list/weather_protection_types = "ash"
```

Comment:
What did you create this var for? It's never used.

Reply:
good question :p